docs/source/script_tutorial.rst

```diff
@@ -263,15 +278,17 @@ Graphical Tools
         # put control inside Menu
         menu.addLayoutWidget(control)
 
-    The only problem here is that you lose the close button for `control`, to
-    over come that menu has the layout sub-menu that allow the user to close
+    The only problem here is that you lose the close button for `control`. To
+    overcome this deficiency, the menu has a layout sub-menu so the user can close
```

Comment:
Okay, I think I may just remove this. If 2.6 users come up we'll deal with that or as you suggested we could always put a note in the README file.